brianlong/www.validators.app
# 178910745_fix_displayed_validator_software_version_fix

#### What's this PR do?
- Use software_version from score on validator show page

#### How should this be manually tested?
- See the index with software_versions
- See the show page of the one of the validators to see if software version is correctly displayed
- Run the daemons, see if there are no errors

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/178910745

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
